src/RuleCollection.php

```diff
@@ -1494,6 +1496,7 @@ public static function processImportRules()
             if (!$item->isEntityAssign()) {
                 $params['entities_id'] = 0;
             } else {
+                $rule['entities_id'] = $DB->escape(Html::entity_decode_deep($rule['entities_id']));
```

Comment:
I may miss something, but `$rule` object correspond to an entry added in `previewImportRules()`, so it should already be decoded.
```suggestion
                $rule['entities_id'] = $DB->escape($rule['entities_id']);
```